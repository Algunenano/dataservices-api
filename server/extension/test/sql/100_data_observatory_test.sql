SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'obs_get_demographic_snapshot'
              AND oidvectortypes(p.proargtypes)  = 'text, text, geometry, text, text');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'obs_get_segment_snapshot'
              AND oidvectortypes(p.proargtypes)  = 'text, text, geometry, text');