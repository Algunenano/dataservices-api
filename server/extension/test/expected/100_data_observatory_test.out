SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'obs_get_demographic_snapshot'
              AND oidvectortypes(p.proargtypes)  = 'text, text, geometry, text, text');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'obs_get_segment_snapshot'
              AND oidvectortypes(p.proargtypes)  = 'text, text, geometry, text');
 exists 
--------
 t
(1 row)

